src/patcher/java/net/minecraftforge/gradle/patcher/PatcherExtension.java

```diff
@@ -144,7 +144,7 @@ public void setExc(List<File> files) {
         return excs;
     }
 
-    public void runs(Closure<? super RunConfig.Container> value) {
+    public void runs(Closure value) {
         value.setResolveStrategy(Closure.DELEGATE_FIRST);
```

Comment:
Is there a real need to change this? As it'd be better to not have the blank generic.